pnpm-lock.yaml

```diff
@@ -17438,6 +17325,47 @@ packages:
     resolution: {integrity: sha512-V50KMwwzqJV0NpZIZFwfOD5/lyny3WlSzRiXgA0G7VUnRlqttta1L6UQIHzd6EuBY/cHGfwTIck7w1yH6Q5zUw==}
     dev: true
 
+  '@pkg.csb.dev/mui/material-ui/commit/75cd58f2/@mui/internal-test-utils(@babel/core@7.24.5)(@types/react@18.2.60)(react-dom@18.2.0)(react@18.2.0)':
+    resolution: {tarball: https://pkg.csb.dev/mui/material-ui/commit/75cd58f2/@mui/internal-test-utils}
+    id: '@pkg.csb.dev/mui/material-ui/commit/75cd58f2/@mui/internal-test-utils'
+    name: '@mui/internal-test-utils'
+    version: 1.0.0
+    peerDependencies:
+      react: ^18.2.0
+      react-dom: ^18.2.0
+    dependencies:
+      '@babel/plugin-transform-modules-commonjs': 7.24.6(@babel/core@7.24.5)
+      '@babel/preset-typescript': 7.24.6(@babel/core@7.24.5)
+      '@babel/register': 7.24.6(@babel/core@7.24.5)
+      '@babel/runtime': 7.24.6
+      '@emotion/cache': 11.11.0
+      '@emotion/react': 11.11.4(@types/react@18.2.60)(react@18.2.0)
+      '@mnajdova/enzyme-adapter-react-18': 0.2.0(enzyme@3.11.0)(react-dom@18.2.0)(react@18.2.0)
+      '@testing-library/dom': 10.1.0
+      '@testing-library/react': 15.0.7(@types/react@18.2.60)(react-dom@18.2.0)(react@18.2.0)
+      chai: 4.4.1
+      chai-dom: 1.12.0(chai@4.4.1)
+      dom-accessibility-api: 0.6.3
+      enzyme: 3.11.0
+      format-util: 1.0.5
+      fs-extra: 11.2.0
+      jsdom: 24.1.0
+      lodash: 4.17.21
+      mocha: 10.4.0
+      playwright: 1.44.1
+      prop-types: 15.8.1
+      react: 18.2.0
+      react-dom: 18.2.0(react@18.2.0)
+      react-test-renderer: 18.2.0(react@18.2.0)
+      sinon: 15.2.0
```

Comment:
Nitpick: mui-x is on `sinon@v16` already, have you considered bumping it to on monorepo to have it in sync or is there a roadblock for it? 🤔

Reply:
All good here. It turns out it was jsdom's fault.